engine/crash/src/wscript

```diff
@@ -20,7 +20,7 @@ def build(bld):
                           target = 'crashext_null')
 
     if 'android' in bld.env['PLATFORM']:
-        sig_handler = 'backtrace_libunwind.cpp'
```

Comment:
New backtrace unwinder implemented for Android. It uses clang libunwind

Reply:
we use libunwind from NDK on Android and modified libunwind from Google for macos (because it's a bit better and can get symbols from DWARF).
What do you think is the best name `backtrace_libunwind_ndk.cpp`?